build/media-suite_compile.sh

```diff
@@ -582,6 +582,24 @@ if [[ $ffmpeg != "no" ]] && enabled libsoxr; then
     fi
 fi
 
+if [[ $ffmpeg != "no" ]] && enabled libcodec2 && do_pkgConfig "codec2 = 0.7"; then 
+    _check=(libcodec2.a codec2.pc) 
+	if do_wget -h 0695bb93cd985dd39f02f0db35ebc28a98b9b88747318f90774aba5f374eadb2 \
+            "https://freedv.com/wp-content/uploads/sites/8/2017/10/codec2-0.7.tar.xz"; then
+        do_uninstall include/codec2 "${_check[@]}"
+        touch cmake/GetPrerequisites.cmake
+        sed -i 's|include(${CMAKE_SOURCE_DIR}/cmake/GetPrerequisites.cmake)||' cmake/GetDependencies.cmake.in
+        sed -i 's|get_prerequisites(${CODEC2_DLL} _deps 1 0 "" "")||' cmake/GetDependencies.cmake.in
+        do_cmake
+        log make ninja
+        log install ninja install
```

Comment:
I'd rather disable everything that's binaries for standalone=y. In this case it's probably easier to manually install the headers/lib/pkgconfig and then manually install c2{dec,enc,sim}.exe to bin-audio if standalone=y. If someone eventually needs other binaries, install them later when asked.

Reply:
Done.